### PR TITLE
DeviceSettings: Add summary for DC Dimming and HBM

### PR DESCRIPTION
**Pull request by hg8**
Hi,
Quick PR to add description of DC Dimming, and High Brightness Mode.

I think it's important to explain when DC Dimming should be used for since a lot of users are confused about its function and turn it on when not needed.

### CODE REVIEW

**Comment by idoybh** [comment]
Hey, thank you for submitting this PR

File: DeviceSettings/res/values/aosip_strings.xml [hunk 1]
```diff
@@ -17,7 +17,7 @@
     <string name="display_title">Display</string>
     <string name="srgb_mode_summary"></string>
     <string name="srgb_mode_title">sRGB</string>
-    <string name="hbm_mode_summary"></string>
+    <string name="hbm_mode_summary">Increases the maximum brightness of the display to improve visibility under sunlight. Increases power consumption.</string>
```
Comment (idoybh) [inline DeviceSettings/res/values/aosip_strings.xml]:
```suggestion
    <string name="hbm_mode_summary">Light the display in a brighter than the normally allowed maximum brightness. Using for a long period of time may harm the display</string>
```

As power consumption is pretty clear and it doesn't increases the max, it enforces it.

File: DeviceSettings/res/values/aosip_strings.xml [hunk 2]
```diff
@@ -28,7 +28,7 @@
     <string name="panel_category_summary">Modes for panel color correction</string>
     <string name="off_mode_title">Off</string>
     <string name="tile_panel_mode">Panel modes</string>
-    <string name="dc_mode_summary"></string>
+    <string name="dc_mode_summary">Reduces eye strain in low light conditions but may result in reduced image quality and poor color production.</string>
```
Comment (idoybh) [inline DeviceSettings/res/values/aosip_strings.xml]:
```suggestion
    <string name="dc_mode_summary">Useful for people who are sensitive to PWM\nSide effects include eye strain, nausea and migraines.\nEnabling this implies less accurate display colors</string>
```
As it's not a fact it reduces these for everyone, only a small portion are really sensitive to this.
Also it necessarily causes worse colors (so not may) - but it should not affect "quality".

**Review by idoybh — CHANGES_REQUESTED** [review]

**Comment by idoybh** [comment]
What do you think about these @hg8 ?

File: DeviceSettings/res/values/aosip_strings.xml (hunk 1 above)
Comment (idoybh) [reply DeviceSettings/res/values/aosip_strings.xml]: maybe "damage" is better

File: DeviceSettings/res/values/aosip_strings.xml [hunk 3]
```diff
@@ -17,7 +17,7 @@
     <string name="display_title">Display</string>
     <string name="srgb_mode_summary"></string>
     <string name="srgb_mode_title">sRGB</string>
-    <string name="hbm_mode_summary"></string>
+    <string name="hbm_mode_summary">Light the display in a brighter than the normally allowed maximum brightness. Using for a long period of time may harm the display</string>
```
Comment (idoybh) [inline DeviceSettings/res/values/aosip_strings.xml]:
```suggestion
    <string name="hbm_mode_summary">Light the display in a brighter than the normally allowed maximum brightness. Using for a long period of time may damage the display</string>
```

**Comment by hg8** [comment]
LGTM! Thanks for the clarification @idoybh 👍 
Do you want me to merge the suggestions into a single commit?

**Comment by ghost** [comment]
> Do you want me to merge the suggestions into a single commit?

Rather do the changes, save and add the changes, then do:
* `git commit --amend --no-edit`
* `git push -f`

and it should automatically add the commit with the newer hash here (obviously, only if you pushed to the right branch where you made the pull-request from)

**Pull request merged**